examples/lassen.bsub.sh

```diff
@@ -28,6 +28,9 @@ jsrun_cmd="jsrun -g 1 -a 1 -n $nproc"
 # on why this is important
 export XDG_CACHE_HOME="/tmp/$USER/xdg-scratch"
 
+# Fixes https://github.com/illinois-ceesd/mirgecom/issues/292
```

Comment:
Could you add a comment on why this addresses the issue? Is Nvidia's locking code so bad that one rank's cache updates trample on what another expects to find?

Reply:
Also, I spoke too soon (shock).   It does not seem to sufficiently fix the issue.  We're still iterating on it.

Reply:
I think it's the pocl code that trips over itself. It turns out this doesn't fix the issue completely, multiple ranks on a single node still cause failures.

Reply:
> Could you add a comment on why this addresses the issue? Is Nvidia's locking code so bad that one rank's cache updates trample on what another expects to find?

(Kindof) added something.